### DISCUSSION TITLE
How do I download the ferret package

### DISCUSSION THREAD

**Discussion by irena-glass** [idaholab/moose#23101] [Q&A General] [answered]
I am a student. I want to use ferret for phase field simulation, but I have read a lot of posts on the moose website and github and have not been able to find a way to download ferret.

**Comment by mangerij** [comment, marked as answer]
Hi,

Currently Ferret is a private repo and we give access upon request. I am currently in the process of making it fully open source again along with a minimal documentation website but this will take a few more weeks. Please send me an email john.mangeri@list.lu to discuss what you would like to do so I can provide some input files.

p.s. MOOSE Issues are for changes to the code framework (not additional modules).

**Reply by loganharbour** [reply]
> p.s. MOOSE Issues are for changes to the code framework (not additional modules).

Converted to a discussion.

> I am currently in the process of making it fully open source again along with a minimal documentation website but this will take a few more weeks.

Looking forward to this @mangerij! 🥳 open source _all_ the things

**Reply by irena-glass** [reply]
Thank you for your reply. My tutor wants me to try to simulate the domain structure of ferroelectric materials. The matter is not urgent, so there is no problem waiting for a while.

**Comment by mangerij** [comment]
@irena-glass We made the Ferret repo public today as well as updating the website extensively https://mangerij.github.io/ferret/

**Reply by GiudGiud** [reply]
@mangerij please feel free to make a dedicated announcement post and we'll highlight it for a few weeks

Are you on the MOOSE developer's slack? I couldnt find you there but it's an appropriate forum there

**Reply by mangerij** [reply]
@GiudGiud I will do something like this - perhaps at the end of next week. Letting the paper sit on arXiv for a bit :)

I do use Slack frequently - you can send an invite to john.mangeri@uconn.edu cheers
